Common/src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -100,17 +100,17 @@
   "death.attack.zombification.player.item": "%1$s, %3$s kullanan %2$s'den kaçmaya çalışırken çürüdü.",
 
   "power.toomanyorigins.black_thumb.name": "Solduran Dokunuş",
-  "power.toomanyorigins.black_thumb.description": "Diktiğin tüm dünyaya ait bitkiler solar.",
+  "power.toomanyorigins.black_thumb.description": "Diktiğin dünyaya ait bitkiler solar ve büyüyemez.",
   "power.toomanyorigins.scorched_structure.name": "Ruh Gezer",
   "power.toomanyorigins.scorched_structure.description": "Her zaman ekstra bir Ruh Hızı seviyen var.",
   "power.toomanyorigins.soul_shield.name": "Ruh Kalkanı",
-  "power.toomanyorigins.soul_shield.description": "Maximum canının yarısı veya daha azındaysan, etrafında bir ruh kalkanı oluşturursun, bu kalkan seni hasara karşı daha dayanıklı olmanı ve atılabilir nesnelerden etkilenmemeni sağlar.",
-  "power.toomanyorigins.wither_immunity.name": "Wither Bağışıklılığı",
-  "power.toomanyorigins.wither_immunity.description": "Wither efektine bağışıklılığın var.",
+  "power.toomanyorigins.soul_shield.description": "Canın %50 veya aşağısı ise, etrafında bir ruh kalkanı oluşturursun, bu seni hasara karşı dayanıklı olmanı ve atılabilir nesnelerin seni incitmemesini sağlar.",
+  "power.toomanyorigins.wither_immunity.name": "İçli Dışlı",
+  "power.toomanyorigins.wither_immunity.description": "Solma etkisine bağışıksın.",
   "power.toomanyorigins.wither_toxins.name": "Wither Hasarı",
   "power.toomanyorigins.wither_toxins.description": "Yakın dövüş saldırıların kısa süreliğine wither efekti verir.",
   "power.toomanyorigins.weariness.name": "Bitkinlik",
-  "power.toomanyorigins.weariness.description": "Hayvanlar etrafında Bitkin olduğundan, hayvanları çiftleştiremezsin.",
+  "power.toomanyorigins.weariness.description": "Hayvanlar etrafında bitkin olduğundan onları çiftleştiremezsin.",
```

Comment:
idk why weary was capitilazed here, and it now uses they instead of saying animals a second time